books-list.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet"  type="text/css" href="application.css">
+  </head>
+  <body>
+    <div class="list-container">
+      <div class="book-item">
+        <div class="book-cover-mini"></div>
```

Comment:
Change this by a img

Reply:
I made it this way because I thought it is a bad practice to create many `img` tags, trying to load many times the same resource (in this case, the image). So I'm just loading the actual image in a single unique moment (in the `background: url...` style). Are you telling me this because it is bad to define an empty `div` just with styles? If that's so, I changed this in the last commit to be an `<a>` tag, so the unique single load of the image is maintained and this tag has more sense. Tell me what you think about it.

Reply:
It's not really a bad practice at all. You should always use the corresponding tag to what element you need, `img` for images, `a` for links, `h` for titles and so on. The use of images on CSS or HTML only change the way the images are being requested, nothing more (maybe a couple of details but nothing too important 😝 ). Don't be afraid of repeating tags, just be aware to create a good semantic HTML 😄